extensions/ql-vscode/src/view/results/locations/ClickableLocation.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import { useCallback } from "react";
+import { ResolvableLocationValue } from "../../../common/bqrs-cli-types";
+import { jumpToLocation } from "../result-table-utils";
+
+interface Props {
+  loc: ResolvableLocationValue;
+  label: string;
+  databaseUri: string;
+  title?: string;
+  handleClick?: () => void;
+}
+
+/**
+ * A clickable location link.
+ */
+export function ClickableLocation({
+  loc,
+  label,
+  databaseUri,
+  title,
+  handleClick,
+}: Props): JSX.Element {
+  const jumpToLocationHandler = useCallback(
```

Comment:
In reference to my previous comment, this is the one I suggested renaming to `handleClick`.